### PR TITLE
Upgrade to latest tech docs template

### PR DESCRIPTION
**Pull request by tijmenb**
This PR is almost the same as https://github.com/alphagov/reliability-engineering/pull/10 and https://github.com/alphagov/gds-way/pull/132.

This upgrades the tech docs to the latest version, which uses the [govuk_tech_docs gem](https://github.com/alphagov/tech-docs-gem) to pull in CSS, JS, fonts, images and layouts. This means that a lot of files can be removed and only customised code and configuration remains. You can read more about it in https://github.com/alphagov/tech-docs-template/pull/144.

I'm updating projects that haven't upgraded yet as part of the GOV.UK firebreak, in the hope that if we are all on the same version it's more likely that sites will keep up to date and contribute back to the main project.

## Automatic upgrades

Since future upgrades will be done by updating the `govuk_tech_docs` dependency, we can set up [Dependabot](https://dependabot.com) to do the upgrades for us. Whenever a new version of the gem is available, Dependabot will raise a PR to update the version. It looks something like this: https://github.com/alphagov/govuk-developer-docs/pull/858.

To enable this, ask a GitHub owner to add the repo to [repos Dependabot can access](https://github.com/organizations/alphagov/settings/installations/87197). After that, you can go to the [dependabot dashboard and click "Add project"](https://app.dependabot.com/accounts/alphagov/repos) to turn on automatic updating.

## How to test

There should be few changes in the layout or behaviour after this upgrade. I've checked this by comparing https://registers-docs.cloudapps.digital/ to the new site. It looks like there are minor layout differences after this PR, probably caused by updates to the template itself since the last update of this site (more than a year ago).

## Before

screen shot 2018-04-05 at 13 55 07

## After

screen shot 2018-04-05 at 13 55 16

---

Part of the Q4 GOV.UK 🔥 firebreak: https://trello.com/c/QbspvRc2

### CODE REVIEW

**Comment by arnau** [comment]
@bahmady Dependabot configured. Ignore any PR from it until we merge this PR please.

File: Gemfile [hunk 1]
```diff
@@ -8,14 +6,4 @@ gem 'wdm', '~> 0.1.0', platforms: [:mswin, :mingw]
 # Windows does not come with time zone data
 gem 'tzinfo-data', platforms: [:mswin, :mingw, :jruby]
```
[the remark this answers is not in the document]
Comment (tijmenb) [reply Gemfile]: As I understand it, these two are needed for development on Windows and jRuby. They're part of the [default template that ships with middleman](https://github.com/middleman/middleman-templates-default/blame/master/template/Gemfile).

**Comment by bahmady** [comment]
OK... sorry for the late review but LGTM. @arnau do you want to merge this one?

**Comment by arnau** [comment]
On it.

**Pull request closed without merging**